apps/code-examples/src/app/code-examples/lookup/selection-modal/add-item/demo.component.spec.ts

```diff
@@ -55,6 +55,7 @@ describe('Selection modal demo', () => {
 
     TestBed.configureTestingModule({
       imports: [DemoComponent, NoopAnimationsModule],
+      providers: [{ provide: DemoService, useValue: mockSvc }],
```

Comment:
We forgot to provide the mock service, so an artificial delay was getting added to each test, causing sporadic timeouts.